src/electron/future/ipc/story.ts

```diff
@@ -1,28 +1,32 @@
-import { BrowserWindow, ipcMain } from "electron";
+import fsp from "fs/promises";
+
+import { BrowserWindow } from "electron";
 import { OpenAI } from "openai";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Couldn't you just also provide the following with a message?
```suggestion
// @ts-expect-error <message>
```

Reply:
this is still required. `// eslint-disable-next-line @typescript-eslint/ban-ts-comment`